css/main.css

```diff
@@ -117,18 +117,18 @@ div#divHola{
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
 
-div#divServicios{    
-    background-color:#F4F1EF;
+div#div-servicios {    
+    background-color: #F4F1EF;
     display: block;
     text-align: center;
 }
 
-div#divServicios h1{
-    margin-bottom:-160px;
+div#div-servicios h1 {
+    margin-bottom: -160px;
 }
 
-h2#Sverde{
-    color:#00ADBB;
+h2#s-verde {
+    color: #00ADBB;
     height: 42px;
     font-weight: normal;
```

Comment:
You don't need to specified this value, because the `normal` is the default value for this property

Reply:
done